tests/test_ast_symbol_extractor.py

```diff
@@ -215,16 +306,29 @@ def test_undeclared_symbols():
     assert z.imported_symbols == ["numpy"]
     assert z.used_symbols == {"numpy.ones", "twos"}
     assert z.undeclared_symbols == {"twos"}
-    assert z.star_imports == {"abc", "xyz"}
     assert z.symbols == {
+        "*": {
```

Comment:
This is a first pass at a star imports data model.
It's just a set of what was slammed into the model 😄 

Not sure if it had to be a different type, we could jus re-use the "import" type but the purpose is a bit different since the "import" type actually has the symbol exposed in the surface area - "star-import" is more of a warning and will contain a set of imports as opposed to a single import that is "shadowed"